tests/test_pipelines.py

```diff
@@ -771,8 +785,15 @@ def _test_ner_pipeline(
             for key in output_keys:
                 self.assertIn(key, result)
 
-        for ungrouped_input, grouped_result in zip(ungrouped_ner_inputs, expected_grouped_ner_results):
-            self.assertEqual(nlp.group_entities(ungrouped_input), grouped_result)
+        if nlp.grouped_entities:
```

Comment:
conditioned so that grouped_entities=False tests won't fail because of grouped_entities=True

Reply:
LGTM